frontend/src/App.tsx

```
@@ -71,7 +71,12 @@ const App = () => {
                 </AdminClubProvider>
               }
             />
-            <Route path='view-application' element={<ApplicationForm />} />
+            //todo /:clubid로 수정 필요
+            {/*<Route*/}
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)